test/services/optIn/OptInServiceSpec.scala

```diff
@@ -236,30 +236,30 @@ class OptInServiceSpec extends UnitSpec
 
   "OptInService.cumulativeQuarterlyUpdateCounts" should {
 
-    "for proposition with opt-in for only next-tax-year" in {
-
-      val currentOptInTaxYear = CurrentOptInTaxYear(Voluntary, currentTaxYear)
-      val nextOptInTaxYear = NextOptInTaxYear(Annual, nextTaxYear, currentOptInTaxYear)
-      val proposition = OptInProposition(currentOptInTaxYear, nextOptInTaxYear)
-
-      val result = service.getQuarterlyUpdatesCountForOfferedYears(proposition)
-
-      result.futureValue shouldBe QuarterlyUpdatesCountForTaxYearModel(Seq(QuarterlyUpdatesCountForTaxYear(nextTaxYear, 0)))
-    }
-
-    "for proposition with opt-in for current-tax-year" in {
-
-      val currentOptInTaxYear = CurrentOptInTaxYear(Annual, currentTaxYear)
-      val nextOptInTaxYear = NextOptInTaxYear(Voluntary, nextTaxYear, currentOptInTaxYear)
-      val proposition = OptInProposition(currentOptInTaxYear, nextOptInTaxYear)
-
-      when(nextUpdatesService.getQuarterlyUpdatesCounts(ArgumentMatchers.eq(currentOptInTaxYear.taxYear))(any(), any()))
-        .thenReturn(Future.successful(QuarterlyUpdatesCountForTaxYear(currentOptInTaxYear.taxYear, 1)))
-
-      val result = service.getQuarterlyUpdatesCountForOfferedYears(proposition)
-
-      result.futureValue shouldBe QuarterlyUpdatesCountForTaxYearModel(Seq(QuarterlyUpdatesCountForTaxYear(currentOptInTaxYear.taxYear, 1)))
-    }
+//    "for proposition with opt-in for only next-tax-year" in {
+//
+//      val currentOptInTaxYear = CurrentOptInTaxYear(Voluntary, currentTaxYear)
+//      val nextOptInTaxYear = NextOptInTaxYear(Annual, nextTaxYear, currentOptInTaxYear)
+//      val proposition = OptInProposition(currentOptInTaxYear, nextOptInTaxYear)
+//
+//      val result = service.getQuarterlyUpdatesCountForOfferedYears(proposition)
+//
+//      result.futureValue shouldBe QuarterlyUpdatesCountForTaxYearModel(Seq(QuarterlyUpdatesCountForTaxYear(nextTaxYear, 0)))
+//    }
+//
+//    "for proposition with opt-in for current-tax-year" in {
+//
+//      val currentOptInTaxYear = CurrentOptInTaxYear(Annual, currentTaxYear)
+//      val nextOptInTaxYear = NextOptInTaxYear(Voluntary, nextTaxYear, currentOptInTaxYear)
+//      val proposition = OptInProposition(currentOptInTaxYear, nextOptInTaxYear)
+//
+//      when(nextUpdatesService.getQuarterlyUpdatesCounts(ArgumentMatchers.eq(currentOptInTaxYear.taxYear))(any(), any()))
+//        .thenReturn(Future.successful(QuarterlyUpdatesCountForTaxYear(currentOptInTaxYear.taxYear, 1)))
+//
+//      val result = service.getQuarterlyUpdatesCountForOfferedYears(proposition)
+//
+//      result.futureValue shouldBe QuarterlyUpdatesCountForTaxYearModel(Seq(QuarterlyUpdatesCountForTaxYear(currentOptInTaxYear.taxYear, 1)))
+//    }
```

Comment:
Can this commented-it test be removed now we're no longer counting quarterly updates?

Reply:
done